go-server/go/api_default.go

```diff
@@ -0,0 +1,108 @@
+/*
+ * Web Server
+ *
+ * A simple web server for managing users
+ *
+ * API version: 1.0
+ * Contact: dmanor@redhat.com
+ * Generated by: Swagger Codegen (https://github.com/swagger-api/swagger-codegen.git)
+ */
+package swagger
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"path"
+)
+
+type userId int
+
+func AddUser(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json; charset=UTF-8")
+	w.WriteHeader(http.StatusOK)
+
+	err := r.ParseForm()
+
+	if err != nil {
+		log.Fatal(err)
```

Comment:
Nice :) 
Logging is always good for debugging purposes.
maybe it's better to log more detailed error message - see [fmt.Errorf](https://www.geeksforgeeks.org/fmt-errorf-function-in-golang-with-examples/)